builtin/exec_builtin.c

```diff
@@ -24,11 +24,11 @@
 
 #include <nuttx/config.h>
 
-#include <spawn.h>
 #include <fcntl.h>
 #include <errno.h>
 #include <debug.h>
 
+#include "nshlib/nshlib.h"
```

Comment:
nshlib is just a library for the NuttX shell. I think it is not a good idea to reverse-use nshlib's APIs in builtin/system. They are not part of the shell.

Reply:
For example, if the application does not enable nshlib, then it cannot use the system() call? This does not make sense.

Reply:
builtin apps couldn't depend on nshlib, but system already depends on shell since it launches nsh internally, this behavior also explicitly specified in spec:
https://pubs.opengroup.org/onlinepubs/009695399/functions/system.html

Reply:
exec_builtin.c already revert.

system and nsh all use spawn to do background execute.